src/components/ControlledForm/ControlledForm.stories.mdx

```diff
@@ -0,0 +1,119 @@
+# Controlled Form Component
+
+## Type: From
+
+## Controlled Form Props
+
+Although having the nature of a decorator component, the Address Search's usable properties are the ones listed below.
```

Comment:
@higorquintao it seems that there was a mixup in the documentation for the component could you please check? It's also not generating properly.